Categories/Morphism/Isomorphism.agda

```diff
@@ -188,10 +50,7 @@ module _ where
   --
   flip-iso : (f : A ≅ B) {g : A ⇒ C} {h : B ⇒ C} →
              g ≈ h ∘ from f → g ∘ to f ≈ h
-  flip-iso f {g} {h} tr₁ = begin
-    g ∘ to f            ≈⟨ pushˡ tr₁ ⟩
-    h ∘ from f ∘ to f   ≈⟨ elimʳ (isoʳ f) ⟩
-    h                   ∎
+  flip-iso f tr₁ = sym (switch-fromtoʳ f (sym tr₁))
```

Comment:
Hm, this makes me think I should have defined `flip-iso` and `push-eq` in `Categories.Morphism.Reasoning.Iso` instead. Would you mind moving them there and updating the dependencies? Or I could do that in a follow-up PR.

Reply:
OK. I'll do that.